src/contracts/core/StrategyManagerStorage.sol

```diff
@@ -83,5 +81,5 @@ abstract contract StrategyManagerStorage is IStrategyManager {
      * variables without shifting down storage in the inheritance chain.
      * See https://docs.openzeppelin.com/contracts/4.x/upgradeable#storage_gaps
      */
-    uint256[40] private __gap;
+    uint256[42] private __gap;
```

Comment:
not sure why/when this gap size changed, but it looks to me like it should still be 40, not changed to 42?
I ran `forge inspect src/contracts/core/StrategyManager.sol:StrategyManager storage --pretty` and see the gap starting at slot 211, while this contract's storage starts at slot 201 (so to use a total of 50 slots, gap size of 40 seems appropriate).

Reply:
fixed

Reply:
looks like this was addressed in commit https://github.com/Layr-Labs/eigenlayer-contracts/pull/127/commits/feb53c860b20f3d3f164b4359f3ec59989347c9b